src/App.js

```diff
@@ -1,29 +1,10 @@
-import logo from './logo.svg';
+import ReactDOM from 'react-dom';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  ReactDOM.render(<Calculator />,
+    document.getElementById('container'));
```

Comment:
- [ ] App function should be class based please work on that

Reply:
This code is from the develop branch which is not related to the branch I am submitting on this pull request.